tools/V4-CSDL-to-OpenAPI.xsl

```diff
@@ -60,7 +60,8 @@
   <xsl:param name="resourceColor" select="'{bg:dodgerblue}'" />
 
   <xsl:variable name="csdl-version" select="/edmx:Edmx/@Version" />
-  <xsl:variable name="option-prefix">

```

Comment:
How about making this
~~~xml
<xsl:param name="option-prefix" select="'$'" />
~~~
and moving it up to line 61?

And remove the old code, it's in the git history.